R/list-combine.R

```diff
@@ -22,6 +22,7 @@
 #'   same size (i.e. number of rows).
 #' @param name_repair One of `"unique"`, `"universal"`, or `"check_unique"`.
 #'   See [vctrs::vec_as_names()] for the meaning of these options.
+#' @param keep_empty An optional Logical to keep empty elements of a list as NA.
```

Comment:
Could you try rewriting this using the formula "If `FALSE` (the default), then ...; if `TRUE`, then ...`.